### ISSUE TITLE
Loss declines slowly on v100

### ISSUE THREAD

**Issue by dgl547437235** [open-mmlab/mmdetection#8406]
Hi，do you test mmdetection on Tesla V100-SXM2,i train model on 4 v100,but loss declines slowly and shocked ,when i train on 3060,loss declines normal（你们有测试过mmdetection在v100上跑吗，我在一台4核的v100上跑模型，损失下降得很慢并且处于持续震荡，我在3060和2080的机器上训练模型，损失下降得较为正常）

**Comment by BIGWangYuDong** [comment]
maybe your batch size and the learning rate are different? Haven't met this problem before.
